src/TinyHelpers/Extensions/DateTimeExtensions.cs

```diff
@@ -39,4 +39,43 @@ public static TimeOnly ToTimeOnly(this DateTime dateTime)
     public static TimeOnly ToTimeOnly(this TimeSpan timeSpan)
         => TimeOnly.FromTimeSpan(timeSpan);
 #endif
+
+    public static string ToFriendlyDateTimeString(this DateTime Date)
+    {
+        return FriendlyDate(Date) + " @ " + Date.ToString("t").ToLower();
```

Comment:
I have some doubts about providing a method like this. The return format is custom and hard-coded, so probably it will fit only a few scenarios.

Reply:
I added the format as parameter with a default value